spec/support/rack_solr_shared_examples.rb

```diff
@@ -0,0 +1,73 @@
+# frozen_string_literal: true
+
```

Comment:
This is essentially a copy of service_controller_shared_examples.rb, but rather than make assertions about specific states within the rails controller, it makes assertions based the solr queries we expect to send